.github/workflows/markdown-links-check/markdown-links-check-config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "timeout": "5s",
+  "retryCount":10,
+  "aliveStatusCodes": [403, 400, 302, 200, 0]
```

Comment:
This is getting out of sync with #4907.  I'd prefer we focus on #4907, get that merged, then merge that commit to gh-pages with a change for the `base-branch` during that PR.  Doing this as parallel PRs is leading to diverging code and likely one will have bugs/features the other will not.

Reply:
sure, closed this pr,  thanks